lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/GroupCommClient.cs

```diff
@@ -80,7 +80,7 @@ public sealed class GroupCommClient : IGroupCommClient
                     group.WaitingForRegistration();
                 }
             }
-            catch (SystemException e)
+            catch (Exception e)
```

Comment:
You changed exception in OperatorTopology.WaitForTaskRegistration() into  ReefRuntimeException, but here you catch generic Exception. That changed the original meaning. The same type should be used at both throw and catch sides.